client/rest-high-level/src/test/java/org/elasticsearch/client/CrudIT.java

```diff
@@ -855,6 +900,8 @@ public void testDeleteByQuery() throws IOException {
                             .source(Collections.singletonMap("foo", 1), XContentType.JSON))
                         .add(new IndexRequest(sourceIndex, "type", "2")
                             .source(Collections.singletonMap("foo", 2), XContentType.JSON))
+                        .add(new IndexRequest(sourceIndex, "type", "3")
+                                .source(Collections.singletonMap("foo", 3), XContentType.JSON))
```

Comment:
You indent this differently than the thing above it.

Reply:
Non-intentional indenting indeed. As penance I now will have to repeat this rapidly a couple of times. And will change of course.